package.json

```diff
@@ -39,7 +40,12 @@
       "gulp-rename": "^1.2.0",
       "gulp-sequence": "0.4.4",
       "gulp-shell": "^0.5.1",
-      "yeoman-assert": "2.1.1"
+      "mocha": "2.3.4",
+      "yeoman-assert": "2.1.1",
+      "yeoman-test": "1.0.0"
+    },
+    "scripts": {
+        "test": "mocha test/*"
```

Comment:
I think should be ok to keep in both

On Sat, 23 Jan 2016 5:15 pm Pascal Grimaud <notifications@github.com> wrote:

> In package.json
> <https://github.com/jhipster/generator-jhipster-module/pull/11#discussion_r50618178>
> :
>
> > @@ -39,7 +40,12 @@
> >        "gulp-rename": "^1.2.0",
> >        "gulp-sequence": "0.4.4",
> >        "gulp-shell": "^0.5.1",
> > -      "yeoman-assert": "2.1.1"
> > +      "mocha": "2.3.4",
> > +      "yeoman-assert": "2.1.1",
> > +      "yeoman-test": "1.0.0"
> > +    },
> > +    "scripts": {
> > +        "test": "mocha test/*"
>
> Yes, it's in gulpfile.js, but by adding in package.json, tests can be
> launch with npm test too
>
>    - package.json => npm test
>    - gulpfile.js => gulp test Should we keep both ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster-module/pull/11/files#r50618178>
> .
>